### DISCUSSION TITLE
Ethernet Link Class

### DISCUSSION THREAD

**Discussion by finwes** [EIPStackGroup/OpENer#381] [Q&A] [answered]
When I configure -DOPENER_IS_DLR_DEVICE:BOOL=ON I get the following compilation errors.

![image](https://user-images.githubusercontent.com/19223210/175960808-6acd2faf-145b-4f98-b564-ce7ce276a171.png)

What Am I doing wrong? Thank you in advance

**Comment by MartinMelikMerkumians** [comment, marked as answer]
I guess the issue here is, that the code was contributed some time ago, but I don't have a DLR ready device, so it is probably out of date.
If you try to fix it, I am happy to help
